src/SignInUser/SignInUser.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react'; 
+import { shallow } from 'enzyme'; 
+import { SignInUser, mapDispatchToProps } from './SignInUser.js'; 
+import { authenticateUser } from '../Thunks/authenticateUser.js'
+
+describe('SignInUser', () => {
+  let mockUserName; 
+  let mockPassword;
+  let mockEvent
+  let wrapper
+
+  beforeEach(() => {
+    wrapper = shallow(<SignInUser signIn={jest.fn()}/>)
+    mockUserName = 'Rajaa'; 
+    mockPassword = 'password'; 
+    mockEvent = {
+     target: {
+       name: 'userName',
+       value: 'Derek'
+      }
+    }
+  })
+
+  it('should match snapshot', () => {
+    const wrapper = shallow(<SignInUser />)
+
```

Comment:
we can delete this line. we don't need it because we are assigning wrapper on line 13